.github/workflows/build-deploy-pudl.yml

```diff
@@ -70,6 +70,8 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_metadata.outputs.tags }}
           labels: ${{ steps.docker_metadata.outputs.labels }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
What is this doing?

Reply:
This is caching docker build layers in the GH actions cache, since otherwise the runners have to rebuild everything from scratch every time.